actions/scratch/action.yml

```diff
@@ -0,0 +1,76 @@
+name: scratch
+description: test custom action
+inputs:
+  tf_version:
+    default: "CALL:1.6.4"
+  tg_version:
+    default: "CALL:0.53.5"
+runs:
+  using: composite
+  steps:
+    - name: ok
+      shell: bash
+      run: echo ok
+
+    - name: show env
+      shell: bash
+      run: |
+        sort -z /proc/self/environ |
+        tr '\0' '\n'
+
+    - name: show full context
+      shell: bash
+      run: |
+        : <<'EOF'
+        github:   ${{ toJSON(github) }},
+        env:      ${{ toJSON(env) }},
+        job:      ${{ toJSON(job) }},
+        jobs:     # for reusable workflows only
+        steps:    ${{ toJSON(steps) }},
+        runner:   ${{ toJSON(runner) }},
+        strategy: ${{ toJSON(strategy) }},
+        matrix:   ${{ toJSON(matrix) }},
+        inputs:   ${{ toJSON(inputs) }},
```

Comment:
you deleted two lines but i don't know which
please leave a comment behind (as you see here) showing they're skipped (and a note why, if you know)